packages/combobox/src/Combobox/Combobox.styles.ts

```diff
@@ -305,12 +321,12 @@ export const baseInputElementStyle = css`
 export const inputElementThemeStyle: Record<Theme, string> = {
   [Theme.Light]: css`
     &::placeholder {
-      color: ${palette.gray.dark1};
+      color: ${palette.gray.base};
```

Comment:
is this consistent? we could add `placeholder` to our tokens set

Reply:
design wants to hold on adding `placeholder` to the tokens set. In the future, we may consolidate it with the disabled token since they're the same value but going to leave this using the palette in the time being